jira-config/jira-config-platform/src/main/java/org/xwiki/contrib/jira/config/internal/JIRAConfigurationMigrator.java

```diff
@@ -117,43 +118,17 @@ private void migrate(String wikiId)
         DocumentReference migrationDocumentRef =
             new DocumentReference(MIGRATION_STATUS_DOCUMENT_REFERENCE, wikiReference);
 
+        try {
+            checkAndFixRights(wikiReference);
+        } catch (XWikiException e) {
+            logger.error("Can't check or fix rights for JIRA server Basic Auth configuration", e);
```

Comment:
small detail: it might be good to pass the `wikiReference` to the error message for debugging purposes.

This doesn't prevent the PR from being applied.